.github/workflows/goreleaser-publish.yml

```diff
@@ -0,0 +1,58 @@
+name: goreleaser-publish
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+concurrency: goreleaser
```

Comment:
This will contradict with `goreleaser` workflow and will result in one canceling the other.

Reply:
Will they cancel each other out or just queue up? I assumed the latter and figured that's actually sensible.